### PR TITLE
added hyperlink to advisors

### PR DESCRIPTION
**Pull request by a-wong-8**
added hyperlinks to advisors as per request from amil via email.

### CODE REVIEW

File: app/[lang]/advisors/page.tsx [hunk 1]
```diff
@@ -57,32 +58,36 @@ const AdvisorsPage = ({ params: { lang } }: { params: { lang: Locale } }) => {
...
-                  {texts.map((text, i) => (
-                    <div key={i} className="text-[9px] font-thin md:text-lg">
-                      {text}
+            {sectionAdvisors.advisors.map(
+              ({ img, name, role, texts, link }) => (
+                <div
+                  key={name}
+                  className="relative max-w-40 text-center md:w-[375px] md:max-w-[22rem]"
+                >
+                  <Link href={link}>
```
Comment (gevuong) [inline app/[lang]/advisors/page.tsx:67]: For consistency with other external links on the website, can you add a `target="_blank"` attribute so it opens to a new tab?

File: app/i18n/es/people.ts [hunk 2]
```diff
@@ -173,6 +188,7 @@ const people: People = {
           'CEO - The Brand Garden',
           `Master's Organizational Leadership - Henley`,
         ],
+        link: '',
```
Comment (gevuong) [inline app/i18n/es/people.ts:191]: Link missing? https://www.linkedin.com/in/richardnall/

File: app/i18n/ja/people.ts [hunk 3]
```diff
@@ -207,24 +228,28 @@ const people: People = {
...
           'スタンフォード大学電気工学博士',
         ],
+        link: 'https://people.ece.uw.edu/atlas/',
       },
       {
         img: NobuyoOzawa,
         name: '小澤信与',
         role: '日本臨床開発リード',
         texts: ['臨床開発部グループマネージャー'],
+        link: 'https://virufy.org/en/our-org',
```
Comment (gevuong) [inline app/i18n/ja/people.ts:238]: link to this page will break once 6.0 becomes virufy.org, as there isn't an `/our-org` route. I reached out to Moeka from Japan team to see if she has a link to Nobuyoshi's profile.

File: app/i18n/es/people.ts [hunk 4]
```diff
@@ -212,12 +232,14 @@ const people: People = {
           'Professor of Electrical and Computer Engineering - University of Washington',
           'Ph.D. Electrical Engineering, Stanford',
         ],
+        link: 'https://people.ece.uw.edu/atlas/',
       },
       {
         img: NobuyoOzawa,
         name: 'Nobuyo Ozawa',
```
Comment (gevuong) [inline app/i18n/es/people.ts:239]: Based on Figma and https://virufy.org/en/our-org, I think `Nobuyo` should be replaced with `Nobuyoshi`. Double check the other `/people.ts` files as well. If you can also rename the image file to reflect the correct spelling, it would be appreciated (ie. `NobuyoOzawa` to `NobuyoshiOzawa`).

File: app/i18n/ja/people.ts [hunk 5]
```diff
@@ -177,6 +194,7 @@ const people: People = {
           'MOSH副会長、取締役会メンバー',
           'ノースフロリダ大学コンピューティング理事会メンバー',
         ],
+        link: 'https://www.linkedin.com/in/richardnall/?originalSubdomain=uk',
```
Comment (gevuong) [inline app/i18n/ja/people.ts:197]: I think you meant to use https://www.linkedin.com/in/khwajashaik/? Double check the other `/people.ts` files as well.

File: app/i18n/ja/people.ts [hunk 6]
```diff
@@ -168,6 +184,7 @@ const people: People = {
           'CEO - The Brand Garden',
           'Henleyビジネススクール - 組織リーダーシップ修士号',
         ],
+        link: 'https://www.linkedin.com/in/richardnall/?originalSubdomain=uk',
```
Comment (gevuong) [inline app/i18n/ja/people.ts:187]: replace with `https://www.linkedin.com/in/richardnall` for consistency. Double check the other `/people.ts` files as well.

**Review by gevuong — CHANGES_REQUESTED** [review]

File: app/[lang]/advisors/page.tsx (hunk 1 above)
Comment (a-wong-8) [reply app/[lang]/advisors/page.tsx:67]: ok

File: app/i18n/es/people.ts (hunk 2 above)
Comment (a-wong-8) [reply app/i18n/es/people.ts:191]: 👍🏼

File: app/i18n/ja/people.ts (hunk 3 above)
Comment (a-wong-8) [reply app/i18n/ja/people.ts:238]: pending

File: app/i18n/es/people.ts (hunk 4 above)
Comment (a-wong-8) [reply app/i18n/es/people.ts:239]: done

File: app/i18n/ja/people.ts (hunk 5 above)
Comment (a-wong-8) [reply app/i18n/ja/people.ts]: done

File: app/i18n/ja/people.ts (hunk 6 above)
Comment (a-wong-8) [reply app/i18n/ja/people.ts]: ok done

File: app/i18n/es/people.ts [hunk 7]
```diff
@@ -227,18 +249,21 @@ const people: People = {
...
         name: 'Fumihiko Yokota',
         role: 'Public Health Advisor',
         texts: ['Associate Professor of Epidemiology, Kyushu University'],
+        link: 'https://q-aos.kyushu-u.ac.jp/en/staff/1770/',
       },
       {
         img: TamerFahmy,
         name: 'Tamer Fahmy',
         role: 'Middle East & Africa Market Advisor',
         texts: ['Former Dräger/Philips MedTech, MBA, SP Jain School of Global Management, BCS Biomedical Engineering, Cairo University'],
```
Comment (gevuong) [inline app/i18n/es/people.ts:265]: Hm, based on email,

```
Former Dräger/Philips MedTech - Sleep & Respiratory Care
MBA, S P Jain School of Global Management
Bsc Biomedical Engineering, Cairo University
```

it looks like Tamer's description should be 3 separate strings and is missing " - Sleep & Respiratory Care"?
I think "BCS" should be "BSc". Changes made here should also be reflected in the other `people.ts` files in different locales.

Screenshot 2024-12-02 at 8 20 24 PM

**Review by gevuong — CHANGES_REQUESTED** [review]

File: app/i18n/es/people.ts (hunk 7 above)
Comment (a-wong-8) [reply app/i18n/es/people.ts:265]: I remember amil saying it was "too wordy" so I removed it. BSc makes sense.

Comment (a-wong-8) [reply app/i18n/es/people.ts:265]: I add the "Sleep & Respiratory Care" back

File: app/i18n/ja/people.ts (hunk 3 above)
Comment (a-wong-8) [reply app/i18n/ja/people.ts]: Still no response from Moeka. I say just push what we have now and add the link later, if she has one. I don't think that Ozawa has a link.

Comment (gevuong) [reply app/i18n/ja/people.ts]: She provided a link. I added it to recent commit.
